module-system/module-implementations/sov-ibc/src/context.rs

```diff
@@ -203,7 +333,17 @@ where
     }
 
     fn increase_client_counter(&mut self) {
-        todo!()
+        let next_client_counter = self
+            .ibc
+            .client_counter
+            .get(*self.working_set.borrow_mut())
+            .unwrap_or_default()
+            .checked_add(1)
+            .expect("Client counter overflow");
```

Comment:
I think we should change `increase_client_counter()` to return an error in ibc-rs, instead of crashing here

Reply:
https://github.com/cosmos/ibc-rs/pull/858